src/components/shared/NavigationBurger.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-	<a
+	<button
 		class="navigation-burger"
 		:class="{ 'active': active }"
-		role="button"
-		aria-label="menu"
-		aria-expanded="false"
+		aria-controls="main-navigation-items"
+		:aria-label="$t( active ? 'aria_burger_label_open' : 'aria_burger_label_closed' )"
```

Comment:
I'm not entirely happy with the name of the key, as it could be confusing to potential translators. Something like `mobile_menu_open_state_accessibility_label` would be longer, but more descriptive